src/psbt.cpp

```diff
@@ -19,6 +19,14 @@ PartiallySignedTransaction::PartiallySignedTransaction(const CMutableTransaction
     SetupFromTx(tx);
 }
 
+PartiallySignedTransaction::PartiallySignedTransaction(uint32_t version) :
+    m_version(version)
+{
+    if (GetVersion() >= 2) {
+        tx_version = CTransaction::CURRENT_VERSION;
```

Comment:
This is erroring for me: `psbt.cpp:(.text+0x4ca): undefined reference to 'CTransaction::CURRENT_VERSION'`